gravitee-apim-console-webui/src/entities/application/application.ts

```diff
@@ -34,8 +34,14 @@ export interface Application {
   picture?: string;
   background?: string;
   owner?: any;
-  settings?: unknown;
+  settings?: App;
   disable_membership_notifications?: boolean;
   api_key_mode?: ApiKeyMode;
   origin?: string;
 }
+
+export interface App {
```

Comment:
```suggestion
export interface ApplicationSettings {
```
For the subtypes model naming we often apply this naming convention.

Reply:
Done